pygmt/datasets/__init__.py

```diff
@@ -3,6 +3,7 @@
 # Load sample data included with GMT (downloaded from the GMT cache server).
 
 from pygmt.datasets.earth_age import load_earth_age
+from pygmt.datasets.earth_magnetic_anomaly import load_earth_magnetic_anomaly
```

Comment:
Remember to add an entry for `load_earth_magnetic_anomaly` under [`doc/api/index.rst`](https://github.com/GenericMappingTools/pygmt/blob/v0.7.0/doc/api/index.rst)!

Reply:
Done.